pkg/operator/ceph/csi/util.go

```diff
@@ -140,25 +140,20 @@ func getComputeResource(clientset kubernetes.Interface, key string) []k8sutil.Co
 	return resource
 }
 
-func getToleration(clientset kubernetes.Interface, provisioner bool) []corev1.Toleration {
-	// Add toleration if any
-	tolerations := []corev1.Toleration{}
-	var err error
-	tolerationsRaw := ""
-	if provisioner {
-		tolerationsRaw, err = k8sutil.GetOperatorSetting(clientset, controller.OperatorSettingConfigMapName, provisionerTolerationsEnv, "")
-	} else {
-		tolerationsRaw, err = k8sutil.GetOperatorSetting(clientset, controller.OperatorSettingConfigMapName, pluginTolerationsEnv, "")
-	}
+func getToleration(clientset kubernetes.Interface, TolerationsName string, defaultTolerations []corev1.Toleration) []corev1.Toleration {
```

Comment:
nit: parameter names are lower case
```suggestion
func getToleration(clientset kubernetes.Interface, tolerationsName string, defaultTolerations []corev1.Toleration) []corev1.Toleration {
```